ui/appui-react/src/appui-react/configurableui/ConfigurableUiContent.tsx

```diff
@@ -288,3 +300,15 @@ function useVisibleToolSettings(enabled: boolean) {
     });
   }, [enabled]);
 }
+
+function IModelProvider({ children }: React.PropsWithChildren<object>) {
+  const imodel = useActiveIModelConnection();
+  if (!imodel) {
+    return children;
+  }
```

Comment:
This is not great due to the remounts.

Reply:
[Address comments](https://github.com/iTwin/appui/pull/1392/commits/9db1eeee99db10f30995fd5d41507936f255816f)